server/service/src/asset/delete.rs

```diff
@@ -30,11 +33,16 @@ pub fn delete_asset(ctx: &ServiceContext, asset_id: String) -> Result<String, De
                 None,
             )?;
 
+            let _deleted_location = AssetInternalLocationRowRepository::new(&connection)
```

Comment:
Does not doing this cause a problem?
Since we are soft deleting the asset, we should be able to skip this step, although this would make the locations available for other assets...

Reply:
I'm wanting to delete these locations so that when an asset is soft deleted because it will allow locations to be assigned to new assets. 

Otherwise I'm thinking I'll need to add a soft delete to asset locations also. Maybe that is a good idea?
What do you think?